spec/normative-references.html

```diff
@@ -10,7 +10,7 @@ <h1>Normative References</h1>
   </p>
 
   <emu-note>
-    Throughout this document, the phrase &ldquo;es2023, _x_&rdquo; (where x is a sequence of numbers separated by periods) may be used as shorthand for "ECMAScript 2023 Language Specification (ECMA-262 14<sup>th</sup> Edition, sub clause _x_)".
+    Throughout this document, the phrase &ldquo;es2023, _x_&rdquo; (where x is a sequence of numbers separated by periods) may be used as shorthand for "ECMAScript 2023 Language Specification (ECMA-262 14<sup>th</sup> Edition, sub clause _x_)". Where _x_ is followed by more such sequences of period-separated numbers, separated from each other by commas, each such sequence is also a shorthand for the corresponding sub clause of ECMA-262.
```

Comment:
We don't have any existing instances of this, right? Is this just for future-proofing?

Reply:
We do, actually: https://tc39.es/ecma402/#conventions
> Algorithm conventions, as described in es2023, <emu-xref href="#sec-algorithm-conventions"><a href="https://tc39.es/ecma262/#sec-algorithm-conventions">5.2</a></emu-xref>, and the use of <emu-xref href="#sec-algorithm-conventions-abstract-operations"><a href="https://tc39.es/ecma262/#sec-algorithm-conventions-abstract-operations">abstract operations</a></emu-xref>, as described in es2023, <emu-xref href="#sec-type-conversion"><a href="https://tc39.es/ecma262/#sec-type-conversion">7.1</a></emu-xref>, <emu-xref href="#sec-testing-and-comparison-operations"><a href="https://tc39.es/ecma262/#sec-testing-and-comparison-operations">7.2</a></emu-xref>, <emu-xref href="#sec-operations-on-objects"><a href="https://tc39.es/ecma262/#sec-operations-on-objects">7.3</a></emu-xref>, <emu-xref href="#sec-operations-on-iterator-objects"><a href="https://tc39.es/ecma262/#sec-operations-on-iterator-objects">7.4</a></emu-xref>.

Reply:
Great, thanks for pointing it out. Sad that we didn't have this properly specified until now.